components/src/main/res/layout/editable_list_item.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.motion.widget.MotionLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/motionLayout"
+    android:layout_width="match_parent"
+    android:layout_height="32dp"
+    app:layoutDescription="@xml/scene_editing_mode"
+    tools:showPaths="true">
+
+    <TextView
+        android:id="@+id/benefit_name"
```

Comment:
can we name this something more generic? We can be consistent with the design invision file and call it `column_one`